styles/felipe.css

```
@@ -0,0 +1,3 @@
+.section-felipe {
+    color: orange;
+}
```

Deberíamos aplicar salto de línea al final del archivo.

```suggestion
.section-felipe {
    color: orange;
}

```